scripts/apps/vocabularies/controllers/VocabularyEditController.tsx

```diff
@@ -120,6 +189,17 @@ export function VocabularyEditController(
         $scope.errorMessage = null;
         delete $scope.vocabulary['_deleted'];
 
+        if ($scope.vocabulary.field_type === EDITOR_BLOCK_FIELD_TYPE) {
+            $scope.requestEditor3DirectivesToGenerateHtml.forEach((fn) => {
+                fn();
+            });
+            $scope.vocabulary.field_options = $scope.vocabulary.field_options ?? {};
+            $scope.vocabulary.field_options = {
+                ...$scope.vocabulary.field_options,
+                template: $scope.item.fields_meta?.[editorBlockFieldId].draftjsState,
```

Comment:
shouldn't you also save selected formatting options?

Reply:
No, these are saved on change in `$scope.formattingOptionsOnChange`, because I need to reinitialize the editor through createEditorStore.